src/main/riddl/AllIdsProjections/allIdsProjections.riddl

```diff
@@ -9,8 +9,8 @@ result AllOrgIds is {orgs: OrganizationId*}
 query GetAllEventIds is {}
 result AllEventIds {events: EventId*}
 
-query GetAllMemberIds is {}
-result AllMemberIds is {members: MemberId*}
+query GetAllUserIds is {}
+result AllUserIds is {users: UserId*}
```

Comment:
As mentioned in our conversation, the only changes that are needed are these:
* Add a new User entity to represent all users of any role
* Add a "userId" field to Member to link it to the user that has that role
There should be no need to change these query or result names. The projection can just use the "userId" field from the various member instances